docs/advanced_topics/api/v2/extra_usage.rst

```diff
@@ -0,0 +1,86 @@
+=====================================
+Extra information on exposing the API
+=====================================
+
+Images
+------
+
+If you have in your model an image you can expose the image by using the ImageRenditionField.
+:class:`wagtail.images.api.fields.ImageRenditionField`. The ImageRenditionField will return url, width, height.
```

Comment:
If you have in your model an image you can expose the image by using the ImageRenditionField.

-> 
You can expose an Image's rendition, a specific size of a related Image field's image, by using the ImageRenditionField class.